server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -230,6 +230,16 @@ export function configureWebSocketServices(
                 });
             }
 
+            const document = await storage.getDocument(claims.tenantId, claims.documentId);
+            if (document && document.deletionTime) {
+                // Prevent connection to existing documents that have been marked for deletion.
+                // eslint-disable-next-line prefer-promise-reject-errors
+                return Promise.reject({
+                    code: 404,
+                    message: "Trying to connect to deleted document",
+                });
+            }
+
```

Comment:
Lets not introduce a new network call to Alfred right now. Deli will already ignore the ops from any client which means they will never be sequenced. Historian will also reject any summary read/write. So alfred will only give that client an websocket connection but that connection is useless. So we don't need to block alfred.

Reply:
That makes sense. Removed in the latest update.